.github/workflows/fix-pull-request.yml

```
@@ -65,16 +62,16 @@ jobs:
     - run: echo ${{ needs.manual_approval_deploy.outputs.comment }}
     - name: "Set backend tag"
       id: backend_tag
-      run: echo "backend_tag=$(docker container inspect $(docker container ls -aq) --format "{{.Config.Image}}" | grep -m 1 "victorycenterua/backend:" | cut -d ":" -f2)" >> $GITHUB_OUTPUT   
+      run: echo "backend_tag=backend:fix-${GITHUB_REF##*/}.${RUN_NUM}" >> $GITHUB_OUTPUT   
     - name: "Set frontend tag"
       id: frontend_tag
-      run: echo "frontend_tag=$(docker container inspect $(docker container ls -aq) --format "{{.Config.Image}}" | grep -m 1 "victorycenterua/client:" | cut -d ":" -f2)" >> $GITHUB_OUTPUT   
+      run: echo "frontend_tag=$(doScker container inspect $(docker container ls -aq) --format "{{.Config.Image}}" | grep -m 1 "victorycenterua/client:" | cut -d "/" -f2)" >> $GITHUB_OUTPUT   
     - name: Checkout
       uses: actions/checkout@v2
```

actionlint (1.7.9)

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)